script/validate-data/package.json

```diff
@@ -14,7 +14,7 @@
     "typescript": "^3.9.2"
   },
   "dependencies": {
-    "@actions/core": "^1.2.6",
+    "@actions/core": "^1.6.0",
```

Comment:
Is this a needed change for this template? If not, then can you raise it separately?

Reply:
@mvecha Can this change be taken separately?